packages/package-values.yaml

```diff
@@ -28,3 +28,19 @@ managementPackageRepository:
       version: latest
       #! Gets replaced with imgpkg sha256 at build, this should be name:version
       sha256: "featuregates:latest"
+    - name: cliplugins
+      displayName: cliplugins
+      #! Relative path to package bundle
+      path: packages/management/core-management-cliplugins
+      domain: tanzu.vmware.com
+      version: latest
+      #! Gets replaced with imgpkg sha256 at build, this should be name:version
+      sha256: "core-management-cliplugins:latest"
+    - name: standalone-cliplugins
```

Comment:
we need to create a new key `standalonePackageRepository` and this package there

Reply:
updated